chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -14,10 +15,15 @@ class SoftmaxCrossEntropy(function.Function):
     ignore_label = -1
     normalize = True
 
-    def __init__(self, use_cudnn=True, normalize=True, cache_score=True):
+    def __init__(self, use_cudnn=True, normalize=True, cache_score=True,
+                 class_weight=None):
         self.use_cudnn = use_cudnn
         self.normalize = normalize
         self.cache_score = cache_score
+        self.class_weight = class_weight
+        if class_weight is not None:
```

Comment:
How about using `if self.class_weight:` instead of `if self.class_weight is not None:`?